object_tags : {"1": {distance: 3, orientation: 0.524, predicates: {facing: "charger_1"}}}